app/services/backend/cmd/serve/serve.go

```diff
@@ -56,6 +60,18 @@ func NewServeCmd() *cobra.Command {
 			shutdown := make(chan os.Signal, 1)
 			signal.Notify(shutdown, syscall.SIGINT, syscall.SIGTERM)
 
+			// -------------------------------------------------------------------------
+			// Start Debug Service
+
+			go func() {
+				logger.L.InfoContext(ctx, "startup", "status", "debug v1 router started", "host", debugOpts.Addr)
+
+				if err := http.ListenAndServe(debugOpts.Addr, debug.Mux()); err != nil {
```

Comment:
can we use gin for this ?  I think it would be nice to settle on a single http layer/framework

Reply:
why are using gin instead of a native package? It is an extra dependency and ee don't profit from it. But yes I can :-)

Reply:
I think to avoid a lot of boilerplate especially when we would need to add authentication, authorization and what not given the out of the box support for middleware. 

However the point is mainly to settle on one option :)